api/Makefile

```diff
@@ -27,7 +27,7 @@ image:
 push:
 	docker push anmho/happened
 
-watch: db
+watch:
```

Comment:
Remove db dependency since it was causing this to hang. Can possibly add it back.